easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -109,7 +98,7 @@ class EasyConfig(object):
     Class which handles loading, reading, validation of easyconfigs
     """
 
-    def __init__(self, path, extra_options=None, build_specs=None, validate=True, hidden=None):
+    def __init__(self, path, extra_options=None, build_specs=None, validate=True, hidden=None, rawtxt=None):
         """
         initialize an easyconfig.
         @param path: path to easyconfig file to be parsed
```

Comment:
i like how you add arguments without updating the docstring.
but there is something else really wrong if you need this many named args.